internal/auth0/organization/resource_connections_test.go

```diff
@@ -254,6 +374,72 @@ func TestAccOrganizationConnections(t *testing.T) {
 				),
 			},
+			{
+				Config: acctest.ParseTestName(testAccOrganizationConnectionsSetShowAsButton, testName),
+			},
+			{
+				RefreshState: true,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.#", "1"),
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.0.show_as_button", "true"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.0.show_as_button", "true"),
+				),
+			},
+			{
+				Config: acctest.ParseTestName(testAccOrganizationConnectionsClearShowAsButton, testName),
+			},
+			{
+				RefreshState: true,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.#", "1"),
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.0.show_as_button", "false"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.0.show_as_button", "false"),
+				),
+			},
+			{
+				Config: acctest.ParseTestName(testAccOrganizationConnectionsDelete, testName),
+			},
+			{
+				RefreshState: true,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.#", "0"),
+				),
+			},
+			{
+				Config: acctest.ParseTestName(testAccOrganizationConnectionsSetIsSignupEnabled, testName),
+			},
+			{
+				RefreshState: true,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.#", "1"),
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.0.is_signup_enabled", "true"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.0.is_signup_enabled", "true"),
+				),
+			},
+			{
+				Config: acctest.ParseTestName(testAccOrganizationConnectionsClearIsSignupEnabled, testName),
+			},
+			{
+				RefreshState: true,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.#", "1"),
+					resource.TestCheckResourceAttr("data.auth0_organization.org_data", "connections.0.is_signup_enabled", "false"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization_connections.one_to_many", "enabled_connections.0.is_signup_enabled", "false"),
```

Comment:
It would be great to check for all the other bool values in the test, even if we don't set them in the config, as the state will that for us and we wanna ensure it does so correctly.